src/api/app/views/webui/patchinfo/_form.html.haml

```diff
@@ -138,5 +138,7 @@
         %div{style: "margin-left:5px;"}
           = submit_tag 'Save Patchinfo'
         = hidden_field_tag "name", @name
+        - @package_list.each do |package|
+          = hidden_field_tag 'package_list[]', package
```

Comment:
Hm what are you trying to achieve with this? Why do you pass it around if it never changes?

Reply:
If I remember well, the problem behind this is when we modify the patchinfo via the WebUI (not editing the file) and we have defined packages in the file. The form does not have knowledge about packages and when we save it. The packages will be deleted because we are overwriting the whole file.

Reply:
So why don't you handle this in the controller instead of passing this around?

Reply:
@hennevogel 

> So why don't you handle this in the controller instead of passing this around?

It was already handled this way for the `name` field, in line 140. I took the same approach for `package` fields.